starboard/android/shared/audio_track_audio_sink_type.cc

```diff
@@ -356,24 +350,24 @@ int AudioTrackAudioSink::WriteData(JniEnvExt* env,
                                    const void* buffer,
                                    int expected_written_frames,
                                    SbTime sync_time) {
-  int samples_written = 0;
+  int return_value = 0;
   if (sample_type_ == kSbMediaAudioSampleTypeFloat32) {
-    samples_written =
+    return_value =
         bridge_.WriteSample(static_cast<const float*>(buffer),
                             expected_written_frames * channels_, env);
   } else if (sample_type_ == kSbMediaAudioSampleTypeInt16Deprecated) {
-    samples_written = bridge_.WriteSample(static_cast<const uint16_t*>(buffer),
-                                          expected_written_frames * channels_,
-                                          sync_time, env);
+    return_value = bridge_.WriteSample(static_cast<const uint16_t*>(buffer),
+                                       expected_written_frames * channels_,
+                                       sync_time, env);
   } else {
     SB_NOTREACHED();
   }
-  if (samples_written < 0) {
+  if (return_value < 0) {
```

Comment:
The old code uses `samples_written` as variable name, and when it returns a negative value to signal an error, it uses the comment below to explain the situation that a negative value signals an error.  This is ok.
In the new code, `return_value` is used, and no comment (or variable name) signaling that the return value is actually samples written when it's greater than or equal to 0.  Note that this is the majority of the cases.  I'd suggest to revert to the old variable name, or add a comment below (on line 369) mentioning that this is actually the number of frames written.